src/etc/natvis/libstd.natvis

```diff
@@ -41,7 +41,7 @@
           <If Condition="(base.table.ctrl.pointer[i] &amp; 0x80) == 0">
             <!-- Bucket is populated -->
             <Exec>n--</Exec>
-            <Item Name="{((tuple&lt;$T1, $T2&gt;*)base.table.ctrl.pointer)[-(i + 1)].__0}">((tuple&lt;$T1, $T2&gt;*)base.table.ctrl.pointer)[-(i + 1)].__1</Item>
+            <Item>((tuple&lt;$T1,$T2&gt;*)base.table.ctrl.pointer)[-(i + 1)]</Item>
```

Comment:
We should display each key-value pair as a tuple instead of having the key as the item name to allow developers to examine HashMap keys of user-defined types in WinDbg.